lib/steps/copyFiles.js

```diff
@@ -51,7 +51,7 @@ function () {
 
             console.log(_chalk.default.white("To run your dApp, execute:"));
             console.log(_chalk.default.white.bold("1. 'cd ".concat(name, "'")));
-            console.log(_chalk.default.white.bold("2. 'npm start' or 'yarn start'"));
+            console.log(_chalk.default.white.bold("2. 'npm start' or 'yarn start' (use sudo if working on Ubuntu)"));
```

Comment:
This is likely due to an issue with how you installed npm or some other associated CLI program.  /cc @DalderupMaurice @jeroenptrs to confirm, but I believe that needing sudo here is a sign that you have something on your system installed/configured incorrectly.

Reply:
Normally when you create a project with the create-nos-dapp tool, the "npm install" or "yarn install" is done for you. Only thing left is to cd into the folder and start it. 
Could you confirm it fails to automatically install the dependencies? 

Also, I've encountered the problem occasionally with having to do `sudo xxx install` aswell on AWS servers (aka ubuntu) - but I am pretty sure it has something to do with npm not being installed correctly. I'm not sure what exactly goes wrong in these cases.

Reply:
Hello, I also encountered the same problem on Windows 10, if powershell wasn't started with administrative rights "npm start" would fail in the same way, as it did on Ubuntu.

Reply:
Ubuntu environment: Ubuntu 18.04 LTS
Commands that have been run:
```sudo apt-get install npm```
```sudo npm i -g @nosplatform/create-nos-dapp``` (it wouldn't work without sudo, because it wouldn't have the rights to copy inside  /usr/local/lib/node_modules)
```sudo npm start``` ( it wouldn't build without sudo)

Maybe the problem may be that the npm package was installed globally, I haven't try installing the package locally.

Reply:
Can you remove the (use sudo if working on Ubuntu) mentions? here and in `src/steps/copyFiles.js`

Reply:
I have reverted the commit.